go.mod

```diff
@@ -64,4 +82,8 @@ replace github.com/lightningnetwork/lnd/cert => ./cert
 
 replace git.schwanenlied.me/yawning/bsaes.git => github.com/Yawning/bsaes v0.0.0-20180720073208-c0276d75487e
 
+replace github.com/coreos/go-systemd => github.com/coreos/go-systemd/v22 v22.0.0
+
+replace github.com/btcsuite/btcwallet/walletdb => github.com/bhandras/btcwallet/walletdb v0.0.0-20200218161033-f2ed9c1c77a2
```

Comment:
Can point to `btcwalelt` master as your dep PR has been merged in.

Reply:
There's a new PR in `btcwallet` that needs to be merged beforehand to support `Batch` tests.